src/llvmir2hll/graphs/cfg/cfg_traversal.cpp

```diff
@@ -4,6 +4,8 @@
 * @copyright (c) 2017 Avast Software, licensed under the MIT license
 */
 
+#include <stack>
+#include <tuple>
```

Comment:
Nitpick: We split system and user includes with an empty line (see [our coding guidelines](https://github.com/avast-tl/retdec/wiki/Contribution-Guidelines#c)).